packages/react/scripts/scaffold.js

```diff
@@ -59,6 +62,77 @@ const addExports = (name) => {
   return exportString;
 };
 
+const addESMInputs = (name, files) => {
+  try {
+    const indexPath = files.find((file) => file.indexOf('index.ts') >= 1);
+    const targetPath = 'config/esmInput.js';
+    const newConfig = esmInput;
+    const key = `src/components/${name}/index`;
+
+    newConfig[key] = indexPath;
+
+    const exportString = `const esmInput = ${JSON.stringify(newConfig)};\nmodule.exports = esmInput;`;
+
+    prettier
+      .resolveConfigFile(indexPath)
+      .then((config) => prettier.resolveConfig(config))
+      .then((options) => {
+        const formatted = prettier.format(exportString, { ...options, parser: 'babel' });
+
+        fs.writeFileSync(targetPath, formatted);
+      });
+  } catch (error) {
+    exitError(`Failed to add to esmInput.js: ${error}`);
+  }
+};
+
+const createCoreComponentFiles = (templatePath, destination, name) => {
+  const files = fs.readdirSync(templatePath);
+  const nameHyphens = name.replace(/[A-Z]/g, (match, offset) => (offset > 0 ? '-' : '') + match.toLowerCase());
+
+  const newFiles = files.map((file) => {
+    const sourcePath = `${templatePath}${file}`;
+    const targetPath = `${destination}/${file.split('new-component').join(nameHyphens)}`;
+
+    const data = fs
+      .readFileSync(sourcePath, 'utf-8')
+      .split('[-replace-name-capital-]')
+      .join(name)
+      .split('[-replace-name-hyphens-]')
+      .join(nameHyphens);
+
+    fs.writeFileSync(targetPath, data);
+    return targetPath;
+  });
+
+  return newFiles;
+};
+
+const addCoreExports = (name) => {
+  const exportString = `@import url("./${name}/${name}.css");\n`;
+  try {
+    fs.appendFileSync('../core/src/components/all.css', exportString, 'utf-8');
```

Comment:
This also adds the import to the last line.

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1011/commits/49133eb1cad8efa964ceb8c1a6307990c1ab433e